Patch for clanlib-0.7.8-r2 Bug 154513:

Signal handler deinit() was not declared.

This leads to a compilation failure when a release version
rather than a debug version is built (-DNDEBUG).

This patch adds the missing declaration, thus make the
file compile with or without -DNDEBUG.

diff -Naur clanlib-0.7.8-r2.orig/work/ClanLib-0.7.8/Sources/Core/System/Unix/init_linux.cpp clanlib-0.7.8-r2/work/ClanLib-0.7.8/Sources/Core/System/Unix/init_linux.cpp
--- a/Sources/Core/System/Unix/init_linux.cpp	2003-09-19 10:33:02.000000000 +0000
+++ b/Sources/Core/System/Unix/init_linux.cpp	2007-10-26 13:36:39.000000000 +0000
@@ -52,6 +52,7 @@
 	_begin_time = (long) tv.tv_sec*(long) 1000+(long) tv.tv_usec/(long) 1000;
 
 	#ifdef NDEBUG
+	sighandler_t deinit;
 	signal(SIGSEGV, deinit);
 	#endif
 }
